addons/website/models/mixins.py

```diff
@@ -223,11 +223,17 @@ def create_and_get_website_url(self, **kwargs):
         return self.create(kwargs).website_url
 
     def _compute_can_publish(self):
```

Comment:
I guess we should add a `@api.depends_context('uid')` now? Even if those low level access check methods should be correctly cached themself.